python/paddle/fluid/contrib/utils/lookup_table_utils.py

```diff
@@ -80,48 +78,57 @@ def __get_prefetch_op_tuples(main_program):
     return prefetch_op_tuples
 
 
-def convert_dist_to_sparse_program(main_program):
-    if not main_program._distributed_lookup_table:
+def convert_dist_to_sparse_program(progarm):
+    """
+    WARNING: this function will only used for distributed training with distributed lookup table.
```

Comment:
This seems not a warning.

Reply:
used => be used

Reply:
```used => be used```
Use ```WARNING``` because of this function will only be used for distributed lookup table.